src/pages/troubleshooting/troubleshooting-timeouts.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Troubleshooting Timeouts"
+excerpt: "."
+categories: troubleshooting
+slug: timeouts
+toc: true
+---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2018, 2023                -->
+<!--                                                                    -->
+<!-- SPDX-License-Identifier: Apache-2.0                                -->
+<!--                                                                    -->
+
+
+Timeouts can occur during analyze calls if the documents you send to ACD take too long to process.  This can be due to the size of the document or the configuration of the cartridge and its artifacts with the content of the document or a combination of both.  In addition there are various components in the path which can be configured to adjust their timeouts.  
+
+## Openshift Route timeouts
+Openshift uses Routes for ingress access to the ACD or any proxy authentication service you are exposing infront of ACD. The default timeout for this is 30 seconds.  You can adjust this by adding the *haproxy.router.openshift.io/timeout* annotation to the route you are using.  There are other annotations recommended on routes to help balance the traffic across route instances - see the 
+[Manage Access](/security/manage-access/) page for details.
+## Load balancer timeouts
+Ingress controllers on openshift are tied to load balancers that expose the routes to the outside the cluster network.  These are configured in the ingress controller or in the cloud specific load balancer settings.  On AWS OCP if the ingress controller is of type Internal you can set the timeout on the Classic load balancer associated with the domain name in the AWS Console on its Attributes settings.  In OCP 4.11 and later this can be configured in the ingress controller yaml itself - see [ocp docs](https://docs.openshift.com/container-platform/4.11/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.html) for details. 
```

Comment:
"OpenShift" to be consistent with existing documentation.
Drop "to the" in phrase "that expose the routes to the outside the cluster network".
See "OpenShift documentation" instead of "ocp docs".